samples/client/petstore/r_test/R/Order.r

```diff
@@ -63,6 +63,15 @@ Order <- R6::R6Class(
         self$status,
         self$complete
       )
+    },
+    fromJSON = function(OrderJson) {
+      OrderObject <- jsonlite::fromJSON(OrderJson)
+      self$id <- OrderObject$id
+      self$pet_id <- OrderObject$pet_id
+      self$quantity <- OrderObject$quantity
+      self$ship_date <- Date$new()$fromJSON(OrderObject$ship_date)
```

Comment:
Again you need to pass JSON to fromJSON rather than data frame : 

Date$new()$fromJSON(jsonlite::toJSON(OrderObject$ship_date))